python_modules/dagster/dagster/_config/pythonic_config/__init__.py

```diff
@@ -219,15 +246,43 @@ def to_fields_dict(cls) -> Dict[str, DagsterField]:
 
 
 class PermissiveConfig(Config):
+    """Subclass of `Config` that allows arbitrary extra fields. This is useful for
```

Comment:
The Resources API page uses a PyObject to format `Config` - thoughts on doing that here? If it's weird because it's on the same page, can we format this as code instead of italics?

Reply:
yup, fixed